docs/guides/videos.md

```diff
@@ -0,0 +1,22 @@
+# Requirements
+
+Download FFMPEG (GPL Shared Ver) https://github.com/BtbN/FFmpeg-Builds/releases
+
+Install extension-webm fork in CMD/Terminal: haxelib git extension-webm https://github.com/GrowtopiaFli/extension-webm, lime rebuild extension-webm [windows/mac/linux]
```

Comment:
I'd recommend to use kade extension instead of that one.
haxelib git extension-webm https://github.com/KadeDev/extension-webm
BUT AMAZING IDEA BRO!!!